### ISSUE TITLE
Missing files in sdist

### ISSUE THREAD

**Issue by thatch** [quantmind/ccy#16]
It appears that the manifest is missing at least one file necessary to build from the sdist for version 1.1.0. You're in good company, about 5% of other projects updated in the last year are also missing files.

```
+ /tmp/venv/bin/pip3 wheel --no-binary ccy -w /tmp/ext ccy==1.1.0
Looking in indexes: http://10.10.0.139:9191/root/pypi/+simple/
Collecting ccy==1.1.0
  Downloading http://10.10.0.139:9191/root/pypi/%2Bf/40b/54135fb2edcf0/ccy-1.1.0.tar.gz (20 kB)
    ERROR: Command errored out with exit status 1:
     command: /tmp/venv/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-wheel-edw6af7k/ccy/setup.py'"'"'; __file__='"'"'/tmp/pip-wheel-edw6af7k/ccy/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-wheel-edw6af7k/ccy/pip-egg-info
         cwd: /tmp/pip-wheel-edw6af7k/ccy/
    Complete output (9 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-wheel-edw6af7k/ccy/setup.py", line 32, in <module>
        install_requires = requirements("dev/requirements.txt")[0]
      File "/tmp/pip-wheel-edw6af7k/ccy/setup.py", line 18, in requirements
        for line in read(name).split("\n"):
      File "/tmp/pip-wheel-edw6af7k/ccy/setup.py", line 10, in read
        with open(filename, encoding="utf8") as fp:
    FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-wheel-edw6af7k/ccy/dev/requirements.txt'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by lsbardel** [comment]
This should have been fixed in version 1.2.0, thanks for reporting

**Comment by thatch** [comment]
Confirmed, thanks!
